### PR TITLE
add Ruby 2.5, remove Ruby 2.2

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: moduleroot/.travis.yml.erb [hunk 1]
```diff
@@ -4,9 +4,9 @@
 <% if @configs['test_all_rvms'] -%>
 rvm:
   - 2.1.9
-  - 2.2.6
   - 2.3.0
   - 2.4.1
+  - 2.5.1
```
Comment (ekohl) [inline moduleroot/.travis.yml.erb]: Adding this here and then excluding it in the matrix is redundant.

Comment (mmoll) [reply moduleroot/.travis.yml.erb]: changed.

File: moduleroot/.travis.yml.erb [hunk 2]
```diff
@@ -29,6 +28,8 @@ matrix:
 <% end -%>
     - rvm: 2.4.1
       env: PUPPET_VERSION=5.0
+    - rvm: 2.5.1
```
Comment (ekohl) [inline moduleroot/.travis.yml.erb]: Should this be hidden behind `@configs['test_all_rvms']` so if it's false we still only with the Puppet bundled versions?

Comment (mmoll) [reply moduleroot/.travis.yml.erb]: changed.

File: moduleroot/.travis.yml.erb [hunk 3]
```diff
@@ -26,6 +25,8 @@ matrix:
 <% unless @configs['test_all_rvms'] -%>
     - rvm: 2.1.9
       env: PUPPET_VERSION=4.9
+    - rvm: 2.5.1
```
Comment (ekohl) [inline moduleroot/.travis.yml.erb]: Isn't this logically reversed? This should only be here if `test_all_rvms` is true.

Comment (mmoll) [reply moduleroot/.travis.yml.erb]: Hm, I don't get how that should work... If `test_all_rvms` is false only AIO configs (Puppet 4 with Ruby 2.1 and Puppet 5 with Ruby 2.4) should be tested, correct?

Comment (mmoll) [reply moduleroot/.travis.yml.erb]: OK, I think I got it now

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**